Documentation/6.0.2/README.rst

```diff
@@ -0,0 +1,1518 @@
+* For the Nuage and Puppet modules, go to http://git.openstack.org/cgit/openstack/tripleo-heat-templates/tree/puppet .
+* For the files and script to generate the CMS ID, go to https://github.com/nuagenetworks/nuage-ospdirector/tree/OSPD13/generate-cms-id .
+
+.. Note::  **Please contact Nuage for Nuage Ironic Integration**
+
+Before the Deployment Process
+------------------------------
+
+.. Note:: Before performing the procedures in this document, read the *Director Installation and Usage* guide for OSPD 13: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage .
+
+Create seperate repositories for the following packages:
+
+    * OSC and VRS: `OSC and VRS Packages`_
+    * 6WIND and AVRS ( Only for AVRS Deployment) : `6WIND and AVRS Packages`_
+
+
+OSC and VRS Packages
+~~~~~~~~~~~~~~~~~~~~~~
+
+    * Nuage-bgp
+    * Nuage-metadata-agent
+    * Nuage-openstack-heat
+    * Nuage-openstack-horizon
+    * Nuage-openstack-neutron
+    * Nuage-openstack-neutronclient
+    * nuage-openvswitch (VRS)
+    * nuage-puppet-modules (Latest version 5.1.0)
+    * Nuage-topology-collector
+    * Selinux-policy-nuage
+
+
+6WIND and AVRS Packages
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    * 6windgate-dpdk
+    * 6windgate-dpdk-pmd-mellanox-rdma-core
+    * 6windgate-dpdk-pmd-virtio-host
+    * 6windgate-fp
+    * 6windgate-fpn-sdk-dpdk
+    * 6windgate-fp-ovs
+    * 6windgate-linux-fp-sync
+    * 6windgate-linux-fp-sync-fptun
+    * 6windgate-linux-fp-sync-ovs
+    * 6windgate-linux-fp-sync-vrf
+    * 6windgate-product-base
+    * 6windgate-tools-common-libs-daemonctl
+    * 6windgate-tools-common-libs-libconsole
+    * 6windgate-tools-common-libs-pyroute2
+    * 6wind-openstack-extensions
+    * dkms
+    * nuage-metadata-agent (from el7-6wind)
+    * nuage-openvswitch (from el7-6wind)
+    * python-pyelftools
+    * selinux-policy-nuage-avrs
+    * virtual-accelerator-base
+
+
+Deployment Process
+-------------------
+
+Phase 1: Install OpenStack director on the Undercloud system.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Follow the steps in https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/installing-the-undercloud .
+
+When obtaining images for the Overcloud nodes, replace the upstream Overcloud image with one modified to include Nuage components from Step 2 in this workflow.
+
+If you want to use a remote registry for the Overcloud container images, you need to add the iptables rule on Director (Undercloud Machine) with the Undercloud IP address or interface with external connectivity for NAT. In the below example, the Undercloud IP address is 192.168.24.1 and the external interface name is eth0:
+
+::
+
+    sudo iptables -A POSTROUTING -t nat -s 192.168.24.1/24 -j MASQUERADE
+    (or)
+    sudo iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
+
+
+Phase 2: Configure the basic Overcloud.
```

Comment:
Revise the text as follows.
From:
Configure the basic Overcloud.
To:
Configure the Basic Overcloud